docs/ci-cd-environment/machine-types.md

```diff
@@ -65,6 +65,24 @@ Implementation details:
 2. `Memory` is implemented as DDR4 RAM.
 3. `Disk` is implemented using NvME storage.
 
+### R1 Generation
+
+!!! plans "Available on: <span class="plans-box">Startup</span> <span class="plans-box">Scaleup</span>"
+
+!!! warning "`r1` machines are in Technical Preview stage, please contact support if you are interested to try them."
```

Comment:
Sounds better:
`r1` machines are currently in the Technical Preview stage. If you're interested in trying them out, please contact our support team.